docs/user-guide/migrating-to-2.0.0.md

```diff
@@ -47,10 +47,11 @@ module.exports = {
 };
 ```
 
-## Configuration cascading changes
+## Configuration Cascading Changes
 
-If you previously relied on the fact that ESLint will merge configurations from `.eslintrc` and `package.json` files located in the same directory you will have to choose either `.eslintrc` or `package.json` file
-and move all of your configuration into one or the other. In 2.0.0 `package.json` will be treated just like any other configuration file and will have the lowest priority.
+Prior to 2.0.0, if a directory contained both an `.eslintrc` file and a `package.json` file with ESLint configuration information, the settings from the two files would be merged together. In 2.0.0, only the settings from the `.eslintrc` file are used and the ones in `package.json` are ignored when both are present. Otherwise, `package.json` can still be used with ESLint configuration, but only if no other `.eslintrc` files are present.
```

Comment:
"but only if no other `.eslintrc.*` files are present.` is probably a bit clearer.